code/modules/research/designs/weapon_designs.dm

```diff
@@ -280,7 +280,7 @@
 
 /datum/design/commanderammo
 	name = "Commander magazine (9mm)"
-	desc = "A single stack M1911 reproduction magazine, modified to chamber 9mm and fit into Commander sidearms."
+	desc = "A single stack Candor reproduction magazine, modified to chamber 9mm and fit into Commander sidearms."
```

Comment:
We can probably just rewrite this description entirely considering the Commander and Candor have both been redesigned since the original description here and they aren't actually related anymore.